Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -306,7 +306,7 @@
   description: bounty-description-pancake
   entries:
   - name: bounty-item-pancake
-    amount: 13
```

Comment:
its a bakers dozen

Reply:
A bakers dozen of pancakes requires more than an entire carton of eggs to make. If it's an actual dozen the chef can still produce them in a reasonable time by using their entire supply of eggs and satisfy the bounty. This one is very difficult to pull off normally at 13.